builder/azure/arm/config.go

```diff
@@ -114,6 +116,7 @@ func (c *Config) toTemplateParameters() *TemplateParameters {
 		ImagePublisher:             &TemplateParameter{c.ImagePublisher},
 		ImageSku:                   &TemplateParameter{c.ImageSku},
 		ImageVersion:               &TemplateParameter{c.ImageVersion},
+		ImageUri:                   &TemplateParameter{c.ImageUri},
```

Comment:
If you move this into the switch statement below you save some edits against the KeyVault and Windows JSON.